modules/videoio/src/cap_msmf.cpp

```diff
@@ -1269,26 +1831,28 @@ double CvCapture_MSMF::getProperty( int property_id ) const
         case CV_CAP_PROP_CONVERT_RGB:
                 return convertFormat ? 1 : 0;
         case CV_CAP_PROP_SAR_NUM:
-                return captureFormat.aspectRatioNum;
+                return captureVideoFormat.aspectRatioNum;
         case CV_CAP_PROP_SAR_DEN:
-                return captureFormat.aspectRatioDenom;
+                return captureVideoFormat.aspectRatioDenom;
         case CV_CAP_PROP_FRAME_WIDTH:
-            return captureFormat.width;
+            return captureVideoFormat.width;
         case CV_CAP_PROP_FRAME_HEIGHT:
-            return captureFormat.height;
+            return captureVideoFormat.height;
         case CV_CAP_PROP_FOURCC:
-            return captureFormat.subType.Data1;
+            return captureVideoFormat.subType.Data1;
         case CV_CAP_PROP_FPS:
-            return captureFormat.getFramerate();
+            return captureVideoFormat.getFramerate();
         case CV_CAP_PROP_FRAME_COUNT:
             if (duration != 0)
-                return floor(((double)duration / 1e7)* captureFormat.getFramerate() + 0.5);
+                return floor(((double)duration / 1e7)* captureVideoFormat.getFramerate() + 0.5);
             else
                 break;
         case CV_CAP_PROP_POS_FRAMES:
-            return floor(((double)sampleTime / 1e7)* captureFormat.getFramerate() + 0.5);
+            return floor(((double)sampleTime / 1e7)* captureVideoFormat.getFramerate() + 0.5);
```

Comment:
Use direct frame counter by counting of grab() / ReadSample() calls.

Update also `::setTime()` method. Seek to zero should be accurate, others are not (perhaps we should not return property value after that)